ci/setup_prow_env.sh

```diff
@@ -28,23 +28,28 @@ if [ -z "${OCM_TOKEN:-}" ]; then
     error_exit "missing mandatory variable \$OCM_TOKEN"
 fi
 
-
 CLUSTER_NAME_FILE="${SHARED_DIR}/cluster-name"
 if [[ ! -f "${CLUSTER_NAME_FILE}" ]]; then
     echo "rhcsci-$(mktemp -u XXXXX | tr '[:upper:]' '[:lower:]')" > "${CLUSTER_NAME_FILE}"
 fi
 CLUSTER_NAME=$(cat "${CLUSTER_NAME_FILE}")
 
-CLOUD_PROVIDER_REGION=${LEASED_RESOURCE}
+#Expose aws credentials as explicit TF_VAR format, and do not use .awscred file
+#Take it from prow vault mounted at /var/run/..
+# TF_VAR_aws_access_key=$(cat /var/run/aws-key/aws-key)
+# export TF_VAR_aws_access_key
+# TF_VAR_aws_secret_key=$(cat /var/run/aws-secret/aws-secret)
+# export TF_VAR_aws_secret_key
```

Comment:
Why do you have this comment?
What is preventing using this now?
As long we keep use the .awscre, we actually use the admin creds?

Reply:
Since the minimal has been deleted, and until it is recreated and stable, I can put this code back.
Currently this PR makes sure, ci's main.tf include the implicit credentials way.